components/Tools/ArtistsTable.tsx

```diff
@@ -1,5 +1,5 @@
 import { useToolCallProvider } from "@/providers/ToolCallProvider";
-import { ArtistRecord } from "@/types/Artist";
+import { ARTIST_INFO } from "@/types/Artist";
```

Comment:
why change the name from `ArtistRecord` to `ARTIST_INFO`?
- which naming convention are we using in our app?
- please maintain `consistent naming conventions`